src/pages/PoolsPage/v3/SupplyLiquidityV3/index.tsx

```diff
@@ -0,0 +1,512 @@
+  //   (_step) => {
+  //     history.push(`/add/${currencyIdA}/${currencyIdB}/${_step}`);
+  //   },
+  //   [currencyIdA, currencyIdB, history],
+  // );
+
+  const handlePriceFormat = useCallback((priceFormat: PriceFormats) => {
+    setPriceFormat(priceFormat);
+  }, []);
+
+  function resetState() {
+    dispatch(updateSelectedPreset({ preset: null }));
+    dispatch(setInitialTokenPrice({ typedValue: '' }));
+    dispatch(setInitialUSDPrices({ field: Field.CURRENCY_A, typedValue: '' }));
+    dispatch(setInitialUSDPrices({ field: Field.CURRENCY_B, typedValue: '' }));
+    onStartPriceInput('');
+  }
+
+  const stepLinks = useMemo(() => {
+    const _stepLinks = [
+      {
+        link: 'select-pair',
+        title: `Select a pair`,
+      },
+    ];
+
+    if (mintInfo.noLiquidity && baseCurrency && quoteCurrency) {
+      _stepLinks.push({
+        link: 'initial-price',
+        title: `Set initial price`,
+      });
+    }
+
+    _stepLinks.push(
+      {
+        link: 'select-range',
+        title: `Select a range`,
+      },
+      {
+        link: 'enter-amounts',
+        title: `Enter amounts`,
+      },
+    );
+    return _stepLinks;
+  }, [baseCurrency, quoteCurrency, mintInfo]);
+
+  const stepPair = useMemo(() => {
+    return Boolean(
+      baseCurrency &&
+        quoteCurrency &&
+        mintInfo.poolState !== PoolState.INVALID &&
+        mintInfo.poolState !== PoolState.LOADING,
+    );
+  }, [baseCurrency, quoteCurrency, mintInfo]);
+
+  const stepRange = useMemo(() => {
+    return Boolean(
+      mintInfo.lowerPrice &&
+        mintInfo.upperPrice &&
+        !mintInfo.invalidRange &&
+        account,
+    );
+  }, [mintInfo]);
+
+  const stepAmounts = useMemo(() => {
+    if (mintInfo.outOfRange) {
+      return Boolean(
+        mintInfo.parsedAmounts[Field.CURRENCY_A] ||
+          (mintInfo.parsedAmounts[Field.CURRENCY_B] && account),
+      );
+    }
+    return Boolean(
+      mintInfo.parsedAmounts[Field.CURRENCY_A] &&
+        mintInfo.parsedAmounts[Field.CURRENCY_B] &&
+        account,
+    );
+  }, [mintInfo]);
+
+  const stepInitialPrice = useMemo(() => {
+    return mintInfo.noLiquidity
+      ? Boolean(+startPriceTypedValue && account)
+      : false;
+  }, [mintInfo, startPriceTypedValue]);
+
+  const steps = useMemo(() => {
+    if (mintInfo.noLiquidity) {
+      return [stepPair, stepInitialPrice, stepRange, stepAmounts];
+    }
+
+    return [stepPair, stepRange, stepAmounts];
+  }, [stepPair, stepRange, stepAmounts, stepInitialPrice, mintInfo]);
+
+  const completedSteps = useMemo(() => {
+    return Array(currentStep).map((_, i) => i + 1);
+  }, [currentStep]);
+
+  const [allowedSlippage] = useUserSlippageTolerance();
+  const allowedSlippagePercent: Percent = useMemo(() => {
+    return new Percent(JSBI.BigInt(allowedSlippage), JSBI.BigInt(10000));
+  }, [allowedSlippage]);
+
+  const hidePriceFormatter = useMemo(() => {
+    return true;
+    // if (stepInitialPrice && currentStep < 2) {
```

Comment:
remove dead code

Why do we event need hidePriceFormatter if it alwats returns true?